src/Core/src/Core/IViewBackgroundImagePart.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.Maui
+{
+	public interface IViewBackgroundImagePart : IImageSourcePart
```

Comment:
Why do we need this type? Can't we just implement the interface directly on the Page type?

Reply:
We don't have a Page Type. We have `IContentView` so we could do it on `IContentView` 

But it's just the idea of having lots of little interfaces that provide a specific behavior. In theory any UIView/AView/FrameworkElement can have a BackgroundImage